verde/tests/test_utils.py

```diff
@@ -174,13 +174,16 @@ def test_build_grid_invalid_names():
     data = np.ones_like(coordinates[0])
     with pytest.raises(ValueError):
         make_xarray_grid(coordinates, data, data_names=["bla_1", "bla_2"])
+    # data_names equal to None
+    with pytest.raises(ValueError):
+        make_xarray_grid(coordinates, data, data_names=None)
```

Comment:
Might want to change the docstring here since `data_names` can't be `None`: https://github.com/fatiando/verde/pull/295/files#diff-948da2a0e34f8cfb6c9d6b6498ec131890d1e637c8128eed5629fb5ce854dd7bR252

Reply:
Nice catch! Even `dims` cannot be None!